packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md

```diff
@@ -109,16 +109,22 @@ Rate limits control access to Advanced Server Access APIs by measuring the rate
 
 Every API response includes headers related to rate limiting:
 
-* `X-RateLimit-Limit` - Describes the total number of operations permitted per rate limit period.
-* `X-RateLimit-Remaining` - Describes the total number of remaining operations permitted during the current rate limit period.
-* `X-RateLimit-Reset` - Contains a unix timestamp indicating when the number of rate limit requests fully replenishes.
-* `X-RateLimit-Retry-At` - Sent after exceeding the rate limit and contains a unix timestamp indicating when the number of rate limit requests partially replenishes. If multiple requests share the same timestamp, sending them at the same time may cause additional rate limiting.
+* `X-RateLimit-Limit`: Describes the total number of operations permitted per rate limit period.
+* `X-RateLimit-Remaining`: Describes the total number of remaining operations permitted during the current rate limit period.
+* `X-RateLimit-Reset`: Contains a unix timestamp indicating when the number of rate limit requests fully replenishes.
```

Comment:
Should it be UNIX? The specification puts it in all caps: https://unix.org/version3/overview.html. Additionally, Microsoft, although they have both, have UNIX more often. Your thoughts? We have nothing defined and neither does the InfoDev style guide.

```suggestion
* `X-RateLimit-Reset`: Contains a unix timestamp that indicates when the number of rate limit requests fully replenishes.
```

Reply:
Yep, I missed that. I'll change to UNIX. Thanks!